crdt/woot.go

```diff
@@ -38,11 +41,44 @@ var (
 	ErrBoundsNotPresent    = errors.New("subsequence bound(s) not present")
 )
 
-// New returns a initialized document.
+// New returns an initialized document.
 func New() Document {
 	return Document{Characters: []Character{CharacterStart, CharacterEnd}}
 }
 
+// Load reads a text file from disk and converts it into a CRDT document.
+func Load(fileName string) (Document, error) {
+	doc := New()
+	b, err := os.ReadFile(fileName)
```

Comment:
tiny nitpick: `b` doesn't seems to address the usage, I believe `content` would be a better variable name.